### ISSUE TITLE
Getting the order

### ISSUE THREAD

**Issue by herohamp** [Bloxport/Bloxport_Editor#7]
With the current draggables we are using we cannot get the order of the blocks. So we ether need to find a extension for jQuery draggables, find a hacky solution, or make our own draggables script.

**Comment by joker314** [comment]
I prefer doing our own draggables script. But we need to figure out which browsers we are trying to support.

**Comment by herohamp** [comment]
ok

**Comment by theonlygusti** [comment]
Why don't we just build on top of Snap? Or Blockly?

In fact, blockly is already an implementation of our final goal. Why are we trying to reinvent the wheel? Use blockly as a starting point.

**Comment by C0deAMet** [comment]
@theonlygusti because if we use one of them as a base for our project, we won't know how to add extra exports, blocks, etc. Doing it this way, we will know what to do when we need to implement something.

**Comment by herohamp** [comment]
@C0deAMet hacky solutions do work and can work but ok

**Comment by theonlygusti** [comment]
I feel like @C0deAMet doesn't know enough to be managing a team.

You say some of the most ridiculous things, make some of the most ridiculous choices and clearly have no idea what you're doing. Bit mean, but it's true.

**Comment by herohamp** [comment]
^^^

I agree

**Comment by C0deAMet** [comment]
@theonlygusti @herohamp yes I don't know everything about what I am doing, would you guys like to elect someone to be co-manager of the team, I will be co-manager with them, choose who you wish
